lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -882,7 +882,21 @@ class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
     case 3: {
       if (!m_child_tasks_sp) {
         using task_type = decltype(m_task_info.childTasks)::value_type;
-        const std::vector<task_type> &tasks = m_task_info.childTasks;
+        std::vector<task_type> tasks = m_task_info.childTasks;
+
+        // Remove any bogus child tasks.
+        // Very rarely, the child tasks include a bogus task which has an
+        // invalid task id of 0.
+        llvm::remove_if(tasks, [&](auto task_ptr) {
```

Comment:
`remove_if` does not remove anything, you're probably looking for `llvm::erase_if`, or instead calling `std::erase` with the iterator returned by `llvm::return_if`

Reply:
https://llvm.org/doxygen/namespacellvm.html#ac9a7de5a04920954ac964059cfc428ad